app/controllers/campaigns_controller.rb

```diff
@@ -1,14 +1,25 @@
 # frozen_string_literal: true
+include Pagy::Backend
 
 class CampaignsController < ApplicationController
   before_action :set_campaign, only: %i[show update]
+  after_action { pagy_headers_merge(@pagy) if @pagy }
 
   # GET /campaigns
   def index
     @campaigns = valid_campaigns.order(:end_date).all
     json_response(campaigns_json)
   end
 
+  # GET /campaigns/inactive
```

Comment:
I wonder if we should be using query params here instead to filter so that we can query out inactive campaigns, but also filter out active campaigns. Our regular GET endpoint shouldn't return them all

Reply:
Yeah that probably makes more sense than building an entire new route. So if I'm updating it it'd be like:
```rb
# Get campaigns
def index
  if query_param[:inactive]
  # do whatever
  json(response_with_pagination)
  else
  # do the other thing
  json(response_without_pagination)
  end
end
```